<dec f='include/c++/11/bits/exception_ptr.h' l='105' type='void std::__exception_ptr::exception_ptr::exception_ptr(const std::__exception_ptr::exception_ptr &amp; )'/>
<def f='include/c++/11/bits/exception_ptr.h' l='183' ll='189' type='void std::__exception_ptr::exception_ptr::exception_ptr(const std::__exception_ptr::exception_ptr &amp; __other)'/>
<use f='include/c++/11/bits/exception_ptr.h' l='203' u='c' c='_ZNSt15__exception_ptr13exception_ptraSERKS0_'/>
<use f='include/c++/11/bits/nested_exception.h' l='70' u='c' c='_ZNKSt16nested_exception14rethrow_nestedEv'/>
<use f='include/c++/11/bits/nested_exception.h' l='76' u='c' c='_ZNKSt16nested_exception10nested_ptrEv'/>
